src/typescript/services/apiService.ts

```diff
@@ -0,0 +1,24 @@
+class ApiService {
+    private baseUrl: string;
+    private path: string;
+
+    constructor(baseUrl: string, path: string) {
+        this.baseUrl = baseUrl;
+        this.path = path;
+    }
+
+    async get(): Promise<any> {
```

Comment:
Avoid to use any, should consider another way

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!